### ISSUE TITLE
[Accessibility] VoiceOver is not reading the cards properly

### ISSUE THREAD

**Issue by alivascu** [microsoft/BotFramework-WebChat#4050]
We have an S500 Microsoft customer that reported when a VoiceOver app is reading the context of the conversation, it fails to read any kind of cards.

Basically instead of reading the content of the card, it's saying "1 Attachment" and that's it.

1) Is there anything that we can do from our side to fix it? 
2) Or some development needs to be done from your side to read those cards as well properly.

Thanks!

**Comment by srinaath** [comment]
@compulim Can this be clubbed into the accessiblity efforts you are currently undertaking on Webchat?

**Comment by alivascu** [comment]
@compulim Any insights on this? 
Thanks!

**Comment by compulim** [comment]
Will take a look at this during the current accessibility pass.

**Comment by alivascu** [comment]
@compulim Have you found out anything useful yet that I can share with the customer? 
Thanks!

**Comment by alivascu** [comment]
@compulim Any news on this?

Thanks!

**Comment by alivascu** [comment]
@compulim Customer is waiting for an update...
Thanks!

**Comment by lauren-mills** [comment]
@alivascu - can you confirm if William's guidance has resolved this issue?

**Comment by stevkan** [comment]
Closing as resolved.
